### PR TITLE
Token Update

### LINKED ISSUE #383: Update NFToken
Neo added a NonFungibleToken class that implements the NFT standard.

- [x] Rename to `NFToken` to `NonFungibleToken`
- [x] Make sure that neow3j's NFT implementation is still up to date with Neo. Adapt it if needed.
- [x] Consider adding a `TokenState` generics class similar to what is done in `NonFungibleToken` in neo-core.

### PR DESCRIPTION
**Pull request by mialbu**
Closes #383 #408

### CODE REVIEW

**Review by gsmachado — COMMENTED** [review]
Just took a look at what's here so far.
So far so good.

File: contract/src/main/java/io/neow3j/contract/NeoToken.java [hunk 1]
```diff
@@ -193,21 +214,24 @@ public TransactionBuilder unregisterCandidate(ECPublicKey candidateKey) {
...
+     *                                       are not public keys.
      */
     public List<ECPublicKey> getNextBlockValidators() throws IOException {
         return callFunctionReturningListOfPublicKeys(GET_NEXT_BLOCK_VALIDATORS);
     }
 
-    private List<ECPublicKey> callFunctionReturningListOfPublicKeys(String function)
-            throws IOException {
+    private List<ECPublicKey> callFunctionReturningListOfPublicKeys
+            (String function) throws IOException {
```
[the remark this answers is not in the document]
Comment (mialbu) [reply contract/src/main/java/io/neow3j/contract/NeoToken.java]: It's on one line again with line length 100 now.

**Review by gsmachado — APPROVED** [review]
I'm fine merging this. 👍 

@mialbu please, as discussed, create a new issue with the remaining work to be done in the context of #383 (since #408 is already done). 😄

**Comment by mialbu** [comment]
> @mialbu please, as discussed, create a new issue with the remaining work to be done in the context of #383 (since #408 is already done). 😄

Already created in #410 :smiley:

**Pull request merged**